package.json

```diff
@@ -88,11 +90,13 @@
     "@types/react-highlight-words": "^0.16.1",
     "@types/react-redux": "7.1.18",
     "@types/react-plotly.js": "2.5.0"
+    
   },
   "dependencies": {
     "@blueprintjs/core": "^4.2.1",
     "@blueprintjs/icons": "^4.2.4",
     "@lineup-lite/table": "1.6.0",
+    "@storybook/react": "^6.4.19",
```

Comment:
I think this dependency should be a `devDependency` because we do not use the types for the production build.

Reply:
done